Composer/packages/client/src/components/AppUpdater.tsx

```diff
@@ -60,8 +61,15 @@ const dialogContent: Partial<IDialogContentStyles> = {
 
 const { ipcRenderer } = window;
 
-function SelectOption(props) {
-  const { checked, text, key } = props;
+type SelectOptionProps = {
+  checked?: boolean;
+  text: string;
+  key: string;
+};
+
+function SelectOption(props: SelectOptionProps | undefined) {
```

Comment:
This could be desired, but this enforces the caller to pass an argument.

Reply:
This was done to match the type signature of the `onRenderField` prop on a `ChoiceGroup`'s `options` attribute. It's expecting us to handle the "undefined" case even though this function is only used internally to this component and in cases we control completely.